From rpjday@mindspring.com Thu Jan 25 16:35:19 2007
From: "Robert P. J. Day" <rpjday@mindspring.com>
Date: Thu, 25 Jan 2007 19:34:52 -0500 (EST)
Subject: USB: Fix apparent typo CONFIG_USB_CDCETHER.
Cc: Andrew Morton <akpm@osdl.org>, Greg KH <greg@kroah.com>
Message-ID: <Pine.LNX.4.64.0701251932090.16585@CPE00045a9c397f-CM001225dbafb6>



Replace the apparent typo CONFIG_USB_CDCETHER with
CONFIG_USB_NET_CDCETHER.

Signed-off-by: Robert P. J. Day <rpjday@mindspring.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/otg_whitelist.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/core/otg_whitelist.h
+++ gregkh-2.6/drivers/usb/core/otg_whitelist.h
@@ -31,7 +31,7 @@ static struct usb_device_id whitelist_ta
 { USB_DEVICE_INFO(7, 1, 3) },
 #endif
 
-#ifdef	CONFIG_USB_CDCETHER
+#ifdef	CONFIG_USB_NET_CDCETHER
 /* Linux-USB CDC Ethernet gadget */
 { USB_DEVICE(0x0525, 0xa4a1), },
 /* Linux-USB CDC Ethernet + RNDIS gadget */
